src/test/java/hudson/remoting/ExportTableTest.java

```diff
@@ -11,6 +11,7 @@
  */
 public class ExportTableTest extends TestCase {
     public void testDiagnosis() throws Exception {
+        ExportTable.EXPORT_TRACES = true;
```

Comment:
Restore previous value in a `finally` block, or `@After` etc. (See `FlagRule` for JTH-based tests.)